articles/active-directory/app-proxy/application-proxy-integrate-with-sharepoint-server.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Enable remote access to SharePoint - Azure Active Directory Application Proxy
-description: Covers the basics about how to integrate an on-premises SharePoint server with Azure Active Directory Application Proxy.
+description: Covers the basics about how to integrate on-premises SharePoint Server with Azure Active Directory Application Proxy.
```

Comment:
@kenwith Is the product that's referred to SharePoint Server, or should this be "integrate an on-premises server running SharePoint with Azure . . ."?

Thanks.

Reply:
yes, sounds good here, I think it means the product so SharePoint Server sounds right.